scripts/tests/misc.sh

```diff
@@ -23,3 +23,6 @@ do
     echo "TEST $package std,runtime-benchmarks"
   fi
 done
+
+mkdir tmp
```

Comment:
Just to be on the safe side, ensure that `mkdir` does not fail when the folder already exists:
```suggestion
mkdir -p tmp
```

Reply:
addressed in https://github.com/zeitgeistpm/zeitgeist/pull/759/commits/c1d8c5bd9eb2ee86413843aa3c55b4e789557bb9